statsmodels/sandbox/distributions/estimators.py

```diff
@@ -338,13 +338,6 @@ def gmmobjective(params):
 License: BSD
 """
 
-def hess_ndt(fun, pars, args, options):
```

Comment:
What is this function being deleted?

Reply:
Because numdifftools was dropped long ago

Reply:
Is restoring this a blocker?

Reply:
Yes.  IMO, cleaning should be just cleaning unless there is a clear gain from removing code (like it is duplicated or has already been moved from sandbox, like one of your other PRs).

Reply:
restored and green